<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>482</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>811</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.091, y1:0.101636, x2:0.991379, y2:0.977, stop:0 rgba(209, 107, 165, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>821</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.091, y1:0.101636, x2:0.991379, y2:0.977, stop:0 rgba(209, 107, 165, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QLineEdit" name="usernamefeld">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font: 75 16pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordfeld">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font: 75 16pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmpasswordfeld">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
font: 75 16pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 17pt &quot;Ubuntu&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Register a brand new account</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: qlineargradient(spread:pad, x1:0.573, y1:0.289773, x2:1, y2:0, stop:0 rgba(102, 255, 246, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(235, 148, 61, 255));
font: 75 16pt &quot;Ubuntu&quot;;
color:rgb(97, 53, 131)</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="error2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Ubuntu&quot;;color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
